core/build.gradle

```diff
@@ -14,6 +14,15 @@ dependencies {
     implementation group: 'org.javassist', name: 'javassist', version: javassistVersion
     implementation group: 'org.jboss.shrinkwrap', name: 'shrinkwrap-api', version: shrinkWrapVersion
     implementation group: 'org.jboss.shrinkwrap', name: 'shrinkwrap-impl-base', version: shrinkWrapVersion
+    implementation group: 'net.lingala.zip4j', name: 'zip4j', version: zip4jVersion
+
+    implementation group: 'org.codehaus.plexus', name: 'plexus-utils', version: '3.2.1'
+    implementation (group: 'org.jboss.shrinkwrap.resolver', name: 'shrinkwrap-resolver-api-maven-embedded', version: shrinkwrapResolverDepchainVersion) {
+        exclude group: 'org.codehouse.plexus', module: 'plexus-utils'
+    }
+    implementation (group: 'org.jboss.shrinkwrap.resolver', name: 'shrinkwrap-resolver-impl-maven-embedded', version: shrinkwrapResolverDepchainVersion) {
```

Comment:
Does runtimeOnly not work with maven and its broken dependency version resolution?